COVID19/src/screens/Locations.js

```diff
@@ -49,27 +22,35 @@ const Item = ({place, date, time}) => {
 
       <View style={styles.textsContainer}>
         <Text style={styles.textsPlace}>{place}</Text>
-        <Text style={styles.textsDateTime}>
-          on {date} at {time}
-        </Text>
+        <Text style={styles.textsDateTime}>on {date}</Text>
       </View>
     </View>
   );
 };
 
 const Location = ({navigation}) => {
+  const [dataPlaces, setDataPlaces] = useState();
+  useEffect(() => {
+    AsyncStorage.getItem('ScannedPlaces').then((res) => {
+      if (res === null) {
+        AsyncStorage.setItem('ScannedPlaces', JSON.stringify([]));
+      } else {
+        //console.log(JSON.parse(res));
+        setDataPlaces(JSON.parse(res));
+      }
+    });
+  }, []);
+
   return (
     <View style={styles.mainMainConatiner}>
       <View style={styles.mainContainer}>
-        <Text style={styles.mainTitle}>Where have you been</Text>
+        <Text style={styles.mainTitle}>Locations</Text>
```

Comment:
Put all the text strings on a constants file

Reply:
fixed